src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -365,10 +389,23 @@ def calculate_income(self, idx: pd.Index) -> pd.Series:
 
         return income
 
-    def get_business_details(self, idx: pd.Index) -> pd.DataFrame:
-        """Source of the business details pipeline"""
-        pop = self.population_view.get(idx)[["employer_id"]]
-        business_details = pop.join(
+    def get_business_details(self, ids: Union[pd.Index, pd.Series]) -> pd.DataFrame:
+        """Source of the business details pipeline.
+
+        Args:
+            ids (Union[pd.Index, pd.Series]): The state table index (ie simulant
```

Comment:
This will only work if the Series passed in has the name "employer_id", since you don't name the column of the resultant DataFrame.

I suggest adding a note here that the Series is expected to have an index of simulant ids (though obviously this call will still work even if that is not the case).